build-configs.yaml

```diff
@@ -161,6 +161,12 @@ fragments:
   x86_kvm_guest:
     path: "kernel/configs/kvm_guest.config"
 
+  preempt_rt:
+    path: "kernel/configs/preempt_rt.config"
+    configs:
+      - 'CONFIG_EXPERT=y'
+      - 'CONFIG_PREEMPT_RT=y'
+      - 'CONFIG_PREEMPT_RT_FULL=y' # <= v4.19
```

Comment:
I guess it won't hurt to have this config in the fragment with newer kernel, as it should just result in a warning from `merge_config.sh`.  If however we start having strict checks and all the config options must appear in the resulting `.config` then we would need to create a separate fragment, e.g. `preempt_rt_4.19` with that `RT_FULL` config option and the default one without it.